### PR TITLE
feature (refs T30671): cockpit related tasks - adjust planningDocumentCategories - address bugs

### PR DESCRIPTION
**Pull request by MoritzMandler**
Ticket:
https://yaits.demos-deutschland.de/T30671

Description:
regarding MainStory (refs T30524): https://yaits.demos-deutschland.de/T30524
Adds migration to alter the procedure master-templates elements (Planungskategorien) to match the ones received via a cockpit-msg.
regarding (refs TT29526): https://yaits.demos-deutschland.de/T29526
Adjust the map-section shown for a procedure created via cockpit-msg by mapping the given coords to the coords used for our EPSG3857 projection.
regarding (refs TT29527): https://yaits.demos-deutschland.de/T29527
Two tests indicated that the service-type has to be provided as a lower-key string. This should not be necessary but since I am not 100% sure on this one - this got adjusted on BE-side in XtaGisLayerManager.php -> setMessageGisValues()

### Linked PRs (optional)
The changes were made here https://github.com/demos-europe/demosplan/pull/8544 originally and have to be carried over
here.

different affected repos:
https://github.com/demos-europe/demosplan-plugin-xbauleitplanung/pull/2
https://github.com/demos-europe/demosplan-project-bobhh/pull/19

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/DataFixtures/ORM/TestData/LoadProcedureData.php [hunk 1]
```diff
@@ -928,6 +929,13 @@ protected function loadMasterBluePrintFileElements(
             Elements::FILE_TYPE_VERTEILER,
             Elements::FILE_TYPE_NIEDERSCHRIFT_SONSTIGE,
             Elements::FILE_TYPE_SCOPING_PAPIER,
+            // added elements in bobhh master-template
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/DataFixtures/ORM/TestData/LoadProcedureData.php:932]: Better remove project name here

**Review by mussbach — APPROVED** [review]
One minor remark, other than that lgtm, code review only

File: demosplan/DemosPlanCoreBundle/DataFixtures/ORM/TestData/LoadProcedureData.php (hunk 1 above)
Comment (MoritzMandler) [reply demosplan/DemosPlanCoreBundle/DataFixtures/ORM/TestData/LoadProcedureData.php]: done in 2f3653284b4040509cd6998474849bc911f7ec37

within the archiv: da70afc94d0b51b0d732b041d5b70b0e43b9b58f

**Review by AlexMenschoid — APPROVED** [review]
lgtm, code review only

**Pull request merged**